### PR TITLE
fix

### PR DESCRIPTION
**Pull request by iamstarkov**
fixes https://github.com/cssinjs/react-jss/issues/121

### CODE REVIEW

File: src/Background.js [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-    </div>
-  );
-}
+}) => (
+  <div className={`${classes.bg} ${className}`}>
+    {children}
+  </div>
+);
 
-const styles = ({ background: theme }) => {
```
Comment (iamstarkov) [inline src/Background.js]: this is odd

File: src/Background.js [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-  return {
-    root: {
-      composes: 'background',
```
Comment (iamstarkov) [inline src/Background.js]: `composes` doesnt work like this

File: src/Background.js (hunk 1 above)
Comment (iamstarkov) [inline src/Background.js]: because you are getting here a single `theme` argument, not an object

File: src/Theme.js [hunk 3]
```diff
@@ -1,24 +0,0 @@
...
-  background: {},
-};
-
-/**
- * A React Component to supply a custom jss instance for our components and the theme.
- */
-export default function Theme(props) {
-  return (
-    <ThemeProvider theme={theme}>
-      <JssProvider jss={jss}>
```
Comment (iamstarkov) [inline src/Theme.js]: here you passed jss without any plugins, are you sure you wanted that?

File: src/Background.js (hunk 1 above)
Comment (iamstarkov) [inline src/Background.js]: screen shot 2017-07-18 at 6 53 07 pm

versus

screen shot 2017-07-18 at 6 54 01 pm

File: src/Theme.js (hunk 3 above)
Comment (HenriBeck) [inline src/Theme.js]: In my original code, I pass a custom jss instance, but here I just pass a dummy instance for testing.

Comment (iamstarkov) [inline src/Theme.js]: and sorry, i was wrong, its actually jss with plugins https://github.com/cssinjs/react-jss/blob/master/src/jss.js

File: src/Background.js (hunk 2 above)
Comment (HenriBeck) [inline src/Background.js]: According to the [docs](https://github.com/cssinjs/jss-compose#compose-with-global-classes) it does work like this

**Comment by iamstarkov** [comment]
root problem is in JssProvider https://github.com/iamstarkov/react-jss-theming-example/pull/1

File: src/Background.js (hunk 1 above)
Comment (HenriBeck) [inline src/Background.js]: That's a totally legit syntax. In your first example, the variable you would need to access is called theme and not button. I'm simply destructuring and renaming the button property.

Comment (iamstarkov) [inline src/Background.js]: aha

Comment (iamstarkov) [inline src/Background.js]: i see

Comment (iamstarkov) [inline src/Background.js]: you are right https://repl.it/J7Zj/1

File: src/Background.js (hunk 2 above)
Comment (iamstarkov) [inline src/Background.js]: ok, it seems like im wrong all around

Comment (iamstarkov) [inline src/Background.js]: i am very sorry

**Pull request closed without merging**